README.md

```diff
@@ -1,2 +1,54 @@
-# capybara-evaluator-implementation
-Capybara evaluator Action for Tryber projects
+# Capybara Evaluator Action for Tryber projects
+
+Aplicação para teste e correção dos projetos dos estudantes da Trybe utilizando Capybara.
+
+Esta aplicação vai executar os testes a partir do nome do repositório dos estudantes, que deve ser obtido via ENV VAR.
+
+## Guia de instalação do ambiente de desenvolvimento
+
+Para instalar e executar esta aplicação em ambiente de desenvolvimento, siga este [guia](/INSTALLATION.md).
+
+## Inputs
+
+## Outputs
+
+### `evaluation`
+
+Arquivo JSON em formato base64 com a saída avaliada pelo RSpec.
+
+### `result`
+
+Arquivo JSON em formato base64 com o resultado, em notas por requisito, dos testes executados.
+
+## Exemplo de uso
+```yml
+uses: betrybe/capybara-evaluator-action
+```
+
+## Como obter o valor da output do resultado
+```yml
+- name: Capybara evaluator
+  id: evaluator
+  uses: betrybe/capybara-evaluator-action
+- name: Next step
+  uses: another-github-action
+  with:
+    param: ${{ steps.evaluator.outputs.result }}
+```
+
+## Restrições do projeto
+
+O projeto que precisar utilizar essa action deve possuir um arquivo com o nome `requirements_mapping.json` em seu diretório raiz. Esse arquivo deve conter a seguinte estrutura:
+
+```json
+{
+  "test-name-1": 17,
+  "unit test 2 name": 36,
+}
+```
+
+Onde `"test-name-1"` e `"unit test 2 name"` são os nomes/descrições dos testes e, `17` e `36` são os identificadores (IDs) desses requisitos.
```

Comment:
The values are identifying the project requirements, right? Where do we set which requisite is the 17, for instance ?

Reply:
These requirements are also in the Forest Admin database, therefore, this mapping takes the test name and maps it according to its equivalent ID in the database, so it can be recorded properly by the next action.